include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -274,6 +291,17 @@ typename MatType::elem_type CMAES<SelectionPolicyType,
       return overallObjective;
     }
 
+    // Terminate if sigma/sigma0 is above 10^4 * max(D)
```

Comment:
```suggestion
    // Terminate if sigma/sigma0 is above 10^4 * max(D).
```

To be consistent with the style.